config/config-autoscaler.yaml

```diff
@@ -20,12 +20,23 @@ metadata:
 data:
   # Static parameters:
 
-  # Target concurrency is the desired number of concurrent requests for
-  # each pod. This is the primary knob for fast autoscaling which will
-  # try achieve an concurrency per pod of the target
-  # concurrency. Single-concurrency must target a value close to 1.0.
-  multi-concurrency-target: "1.0"
-  single-concurrency-target: "0.9"
+  # The Revision ContainerConcurrency field specifies the maximum number
+  # of requests the Container can handle at once. Container concurrency
+  # target percentage is how much of that maximum to use in a stable
+  # state. E.g. if a Revision specifies ContainerConcurrency of 10, then
+  # the Autoscaler will try to maintain 5 concurrent connections per pod
+  # on average. A value of 0.5 is chosen because the Autoscaler panics
+  # when concurrency exceeds 2x the desired set point. So we will panic
+  # when we reach the limit.
+  container-concurrency-target-percentage: "0.5"
```

Comment:
Does this mean that under stable state we'll create twice as many resources as "necessary"? That feels a little aggressive.

That setting used to be 0.9 for the "defined concurrency" case.

Reply:
Aggreed that this will probably need some tuning.

Do you want to create an issue to figure out the right set-point here, and use something like 0.7 or 0.9 in the meantime, to keep the existing behavior?

Reply:
I've chosen a default of 0.7 (70% resource usage) based on general best-practice and anecdotal experience with serverless workloads (App Engine).  Opened #1956 to tune the value for different workloads.